index.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
+//shell_exec('find ./src -name "*.php" -exec md5sum {} \; > .toolbox.cache');
+
+$files = shell_exec('md5sum -c .toolbox.cache 2>/dev/null');
```

Comment:
I would like to replace `shell_exec` commands with Symfony Process component. Is it possible?

https://symfony.com/doc/current/components/process.html